config/onprem-iso-fcc.yaml

```diff
@@ -30,8 +30,7 @@ systemd:
             [Service]
             Type=forking
             Restart=no
-            ExecStartPre=podman pull --authfile /etc/assisted-service/auth.json rhel8/postgresql-12:latest
-            ExecStart=podman run -dt --pod assisted-installer --env-file /etc/assisted-service/environment --name db rhel8/postgresql-12:latest
+            ExecStart=podman run -dt --pod assisted-installer --env-file /etc/assisted-service/environment --authfile /etc/assisted-service/auth.json --name db rhel8/postgresql-12:latest
```

Comment:
`quay.io/ocpmetal/postgresql-12-centos7`

Reply:
@mhrivnak Which version should the live ISO ignition use? registry.redhat.io/rhel8/postgresql-12:latest or quay.io/ocpmetal/postgresql-12-centos7?

Reply:
Only publicly-accessible images from this repo (centos or UBI base images), then in the product we'll use rhel-based images. That also means we need a way to change the image for the product.

Reply:
Got it. Updated to quay.io/ocpmetal/postgresql-12-centos7.